exodus_lambda/functions/origin_request.py

```diff
@@ -67,6 +66,13 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         uri = self.resolve_aliases(request["uri"])
 
+        self.logger.info(
+            "The request value for request_origin begining is '%s'", request
+        )
+        self.logger.info(
+            "The uri value for request_origin begining is '%s'", uri
+        )
```

Comment:
So, I think you meant `origin_request` instead of `request_origin`and it would be helpful to format the request dictionary a little to make it more readable; `json.dumps(request, indent=4, sort_keys=True)` should be good.

Similar changes would be welcome for the other log messages.
I don't think this is anything to block the PR over, though. We can follow up with another PR for logging improvements.